types/webscreens-window-placement/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "typescript": "^2.2.2"
```

Comment:
Wait what?

Reply:
It was what the definition I was using as a reference had. Typical, it seems to be the only one alphabetically close to my PR that had one and I just happened to pick it.. Removing now